scripts/restore

```diff
@@ -107,14 +107,14 @@ ynh_mysql_connect_as --user=$db_user --password=$db_pwd --database=$db_name < ./
 ynh_script_progression --message="Restoring the systemd configuration..."
 
 ynh_restore_file --origin_path="/etc/systemd/system/$app.service"
-systemctl enable $app.service
+systemctl enable $app.service --quiet
 
 #=================================================
 # INTEGRATE SERVICE IN YUNOHOST
 #=================================================
 ynh_script_progression --message="Integrating service in YunoHost..."
 
-yunohost service add $app --description "$app daemon for Ghost" --log_type "systemd"
+yunohost service add $app --description="$app daemon for Ghost" --log="/var/log/$app/$app.log"
```

Comment:
```suggestion
yunohost service add $app --description "$app daemon for Ghost" --log "/var/log/$app/$app.log"
```
Not sure the `=` will work

Reply:
`=` work! and they arealso  part of *example_ynh*